--- old/bash_completion	2020-07-25 18:25:49.000000000 +1000
+++ new/bash_completion	2020-08-03 09:19:11.842626170 +1000
@@ -2175,6 +2176,10 @@
 __load_completion()
 {
     local -a dirs=(${BASH_COMPLETION_USER_DIR:-${XDG_DATA_HOME:-$HOME/.local/share}/bash-completion}/completions)
+
+    # Add completions relative to anaconda's relocatable install path
+    dirs+=( /opt/anaconda1anaconda2anaconda3/share/bash-completion/completions )
+
     local ifs=$IFS IFS=: dir cmd="${1##*/}" compfile
     [[ -n $cmd ]] || return 1
     for dir in ${XDG_DATA_DIRS:-/usr/local/share:/usr/share}; do
